app/styles/ui/_commit-message-avatar.scss

```diff
@@ -41,6 +44,10 @@
     justify-content: flex-end;
   }
 
+  .secondary-text {
+    color: var(--text-secondary-color);
+  }
```

Comment:
Suggestion from Amanda to make the new text in the popover less prominent.

Reply:
I like this! ✨  ✨